sdks/nodejs/src/agonesSDK.js

```diff
@@ -23,7 +23,20 @@ class AgonesSDK {
 		this.healthStream = undefined;
 		this.emitters = [];
 	}
-	async close(){
+
+	async connect() {
```

Comment:
We should also update the docs.
I will look at generating API docs automatically for Node.js soon (maybe next weekend)

Reply:
Yep - on my todo list.

Reply:
Done!. PTAL.